_drafts/2020-06-10-introduction-to-graphql.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: post
+author: "Pranshu Kharkwal"
+author_url: https://github.com/pranshukharkwal
+title: "Introduction to GraphQL"
+subtitle: "The future of APIs?"
+bg_url: "https://i.imgur.com/4BVtURe.png"
+tags: [home, graphql, api]
+---
+
+If you are a developer, you would be knowing what an API is. At some point in your career, you would have used APIs to get data from some other application. Because let's be honest, [web scraping]({% post_url 2020-05-09-web-scraping %}) doesn't work all the time, plus using APIs is also a lot more convenient.
+But if API is still an alien term for you, then here is what an API means:
+
+    API stands for Application Programming Interface which is way
+    of communication between two different applications.
+    In simpler terms, API is a way of sending data from one application (server) to another application (client)
+And, when we talk of APIs, the first thing which comes to mind is REST API and the reason behind it is that immensely popular.
+Now we are not going to talk about REST API's in this article, but it is important that you need to know about them if you don't already. So here is a [simple guide](https://www.smashingmagazine.com/2018/01/understanding-using-rest-api/) to teach you all you need to know about API, REST, requests, endpoints, etc.
```

Comment:
@aashish-ak this part is not readable upon deployment

Reply:
Thanks @Manav for pointing out, will fix it.

Reply:
Btw, you tagged someone else.
its @manavmehta